content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -159,6 +159,10 @@ For more information on version skews, see:
    ```shell
    sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
    ```
+   Note: If the command fails use below command  
+   ```shell
+   sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://dl.k8s.io/apt/doc/apt-key.gpg
+   ```
```

Comment:
This might be OK, but:
- why is this the right fix? (let's make a note of that)
- should we actually make https://dl.k8s.io/apt/doc/apt-key.gpg the default?

Also, for a note, we prefer to mark it with a shortcode - see
https://kubernetes.io/docs/contribute/style/style-guide/#note